src/app/banking/vouchers/view-voucher.component.html

```diff
@@ -0,0 +1,89 @@
+    [headingActions]="headingActions$ | async">
+
+    <div class="d-flex flex-grow-1 flex-column">
+      <div
+        [class]="(layout.gtsm$ | async) ? 'd-flex flex-row-reverse justify-content-between' : 'd-flex flex-column' ">
+        <div *ngIf="data.type.image"
+          class="image d-flex justify-content-center mt-2 mb-4 align-self-center">
+          <avatar size="160" imageSize="200" useLightbox
+            [image]="data.type.image"></avatar>
+        </div>
+        <div class="d-flex flex-column flex-grow-1">
+          <label-value [label]="i18n.general.type"> {{ data.type.name}}
+          </label-value>
+          <label-value [label]="i18n.voucher.token"> {{ data.token}}
+          </label-value>
+          <label-value [label]="i18n.transaction.amount">
+            {{ data.amount | currency:data.type.configuration.currency }}
+          </label-value>
+          <label-value *ngIf="data.creationType"
+            [label]="i18n.voucher.creationType">
+            {{ data.creationType | capitalizeFirst }}
+          </label-value>
+          <label-value [label]="i18n.general.creationDate">
+            {{ data.creationDate | dateTime}}
+          </label-value>
+          <label-value *ngIf="data.buyer" [label]="i18n.voucher.buy.buyer">
+            <user-link [user]="data.buyer"></user-link>
+          </label-value>
+          <label-value *ngIf="data.buy" [label]="i18n.voucher.buy.payment">
+            <a
+              [routerLink]="['/banking/transfer/' + data.buy.transactionNumber]">
+              {{ data.buy.transactionNumber }}
+            </a>
+          </label-value>
+          <label-value [label]="i18n.general.status">
+            {{ data.status | capitalizeFirst }}
+          </label-value>
+          <label-value *ngIf="data.redeemer"
+            [label]="i18n.voucher.redeem.redeemer">
+            <user-link [user]="data.redeemer"></user-link>
+          </label-value>
+          <label-value *ngIf="data.redeemDate"
+            [label]="i18n.voucher.redeem.date">
+            {{ data.redeemDate | dateTime}}
+          </label-value>
+          <label-value *ngIf="data.redeem"
+            [label]="i18n.voucher.redeem.payment">
+            <a
+              [routerLink]="['/banking/transfer/' + data.redeem.transactionNumber]">
+              {{ data.redeem.transactionNumber }}
+            </a>
+          </label-value>
+          <label-value *ngIf="!data.redeemAfterDateReached"
+            [label]="i18n.general.status">
+            {{ data.redeemAfterDate}}
+          </label-value>
+          <label-value *ngIf="data.expirationDate"
+            [label]="i18n.general.expirationDate">
+            {{ data.expirationDate | dateTime}}
+          </label-value>
+        </div>
+      </div>
+      <!-- Description -->
+      <ng-container *ngIf="data.description"
+        class="d-flex justify-content-start">
+        <h2 class="mt-3"> {{ i18n.general.description }} </h2>
+        <p>
```

Comment:
Voucher type description is a textarea. Line breaks should show line breaks. Hence, add the class `break-nl`.
Also, it is probably best to use a `<div>` instead of `<p>`, because `<p>` adds an extra margin below it.